jerry-math/log.c

```diff
@@ -93,6 +92,7 @@
 double
 log (double x)
 {
+  const double zero = 0.0;
```

Comment:
And in these files, instead of using a zero variable to make division by zero possible on MSVC, the div0 expressions should be changed to +/-INFINITY or NAN. (This is how the patch looked like at some point of time, but then it has changed.)